sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -148,77 +53,42 @@ A configuration setting is the fundamental resource within a configuration store
+### Create a Configuration Client
 
-An application that needs to retrieve startup configurations is better suited using the synchronous client, for example setting up a SQL connection.
+Once you have a connection string you can create a configuration client with either synchronous or asynchronous APIs, as such:
 
-<!-- embedme ./src/samples/java/com/azure/data/appconfiguration/ReadmeSamples.java#L64-L83 -->
+<!-- embedme ./src/samples/java/com/azure/data/appconfiguration/ReadmeSamples.java#L44-L46 -->
 ```Java
 ConfigurationClient configurationClient = new ConfigurationClientBuilder()
     .connectionString(connectionString)
     .buildClient();
-
-// urlLabel is optional
-String url = configurationClient.getConfigurationSetting(urlKey, urlLabel).getValue();
-Connection conn = null;
-try {
-    conn = DriverManager.getConnection(url);
-} catch (SQLException ex) {
-    System.out.printf("Failed to get connection using url %s", url);
-} finally {
-    if (conn != null) {
-        try {
-            conn.close();
-        } catch (SQLException ex) {
-            System.out.printf("Failed to close connection, url %s", url);
-        }
-    }
-}
 ```
 
-An application that has a large set of configurations that it needs to periodically update is be better suited using the asynchronous client, for example all settings with a specific label are periodically updated.
+or
 
-<!-- embedme ./src/samples/java/com/azure/data/appconfiguration/ReadmeSamples.java#L87-L92 -->
+<!-- embedme ./src/samples/java/com/azure/data/appconfiguration/ReadmeSamples.java#L50-L52 -->
 ```Java
 ConfigurationAsyncClient configurationClient = new ConfigurationClientBuilder()
     .connectionString(connectionString)
     .buildAsyncClient();
-
-configurationClient.listConfigurationSettings(new SettingSelector().setLabelFilter(periodicUpdateLabel))
-    .subscribe(setting -> updateConfiguration(setting));
-```
-
-## Examples
-
-The following sections provide several code snippets covering some of the most common configuration service tasks, including:
-
-### Create a Configuration Client
-
-Create a configuration client by using `ConfigurationClientBuilder` by passing connection string.
-
-<!-- embedme ./src/samples/java/com/azure/data/appconfiguration/ReadmeSamples.java#L44-L46 -->
-```Java
-ConfigurationClient configurationClient = new ConfigurationClientBuilder()
-    .connectionString(connectionString)
-    .buildClient();
 ```
 
 ### Create a Configuration Setting
 
-Create a configuration setting to be stored in the configuration store. There are two ways to store a configuration setting:
+Create a configuration setting to be stored in the configuration store.
 
-- `addConfigurationSetting` creates a setting only if the setting does not already exist in the store.
+Use `addConfigurationSetting` to create a setting only if the setting does not already exist in the store:
```

Comment:
nit: should we use the word "insert" rather than create here to reinforce the create only when missing semantics